app/src/main/java/org/oppia/app/home/continueplaying/SectionTitleViewModel.kt

```diff
@@ -0,0 +1,9 @@
+package org.oppia.app.home.continueplaying
+
+import androidx.lifecycle.ViewModel
+
+/** [ViewModel] for section title in [ContinuePlayingFragment]. */
+class SectionTitleViewModel : ContinuePlayingViewModel() {
+  var sectionTitleText = ""
+  var isDividerVisible = false
```

Comment:
Will these two values ever change? If not, can we pass them in the constructor and make them ``val``s, instead? Prefer immutability in all cases where values never need to change.

Reply:
Done. Made val's and used constructor directly.